### PR TITLE
Bugfix/improvements found in 2nd User Testing

### PR DESCRIPTION
**Pull request by anafaraco01**
I corrected several bugfixes and minor improvements based on the feedback gotten.

- [x] The username is displayed on the home page
- [x] Bottom navigation changed, should work just fine
- [x] A message should be displayed whenever there is no category, or habit or thought
- [x] A thought is linked to a user
- [x] Scroll is consistent for categories, habits and thoughts
- [x] Selection of mood, added accent color and scroll.

### CODE REVIEW

**Comment by v-stamenova** [comment]
### Potential issue with longer names
image

**Comment by v-stamenova** [comment]
> I corrected several bugfixes and minor improvements based on the feedback gotten.
> 
> * [x]  The username is displayed on the home page
> * [x]  Bottom navigation changed, should work just fine
> * [x]  A message should be displayed whenever there is no category, or habit or thought
> * [ ]  A thought is linked to a user
> * [ ]  Scroll is consistent for categories, habits and thoughts
> * [ ]  Selection of mood, added accent color and scroll.

"A thought is linked to a user" - I seem unable to replicate this
Issues:

* When I try to select a mood and reason it seems it is not selected (the chosen one does not get darker)
* When I submit a thought after I select the mood and reason, it is not visible in the thoughts page

File: frontend/src/lib/components/MoodItem.svelte [hunk 1]
```diff
@@ -14,14 +14,14 @@
 
     let moodElement = document.getElementById(`moodElement${id}`);
     if (storedMoodId == 0) {
-      moodElement.classList.add("bg-gray-400");
+      moodElement.classList.add("bg-[#fdefc7]");
```
Comment (v-stamenova) [inline frontend/src/lib/components/MoodItem.svelte:17]: Please do not use colors like that. Add them to the tailwind configuration following the convention

File: frontend/src/lib/components/ReasonItem.svelte [hunk 2]
```diff
@@ -14,14 +14,14 @@
 
     let reasonElement = document.getElementById(`reasonElement${id}`);
     if (storedReasonId == 0) {
-      reasonElement.classList.add("bg-gray-400");
+      reasonElement.classList.add("bg-[#fdefc7]");
```
Comment (v-stamenova) [inline frontend/src/lib/components/ReasonItem.svelte:17]: Same thing as the `MoodItem`

File: frontend/src/routes/home/+page.server.js [hunk 3]
```diff
@@ -13,16 +13,16 @@ export const load = async ({ cookies }) => {
     // Get the cookie containing the JWT token
     const jwt = cookies.get('jwt');
 
-    // Send request to the apigateway to check if the user is authenticated
-    const isAuthenticated = await axios.get('http://localhost:3011/verify', {
+    const payload = JSON.parse(Buffer.from(jwt.split('.')[1], 'base64').toString());
```
Comment (v-stamenova) [inline frontend/src/routes/home/+page.server.js:16]: Use `jwt.decode` instead like in `frontend/src/routes/moods/my-mood/[recordId]/thought/+page.server.js`

**Review by v-stamenova — CHANGES_REQUESTED** [review]
For the bigger part it looks good, please resolve the comments I put in this review and check the thread of the PR, I left another comment with an issue that might want our attention. 

Also the last bullet point I could not figure out what to test.

File: frontend/src/lib/components/ReasonItem.svelte (hunk 2 above)
Comment (anafaraco01) [reply frontend/src/lib/components/ReasonItem.svelte:17]: Fixed as well!

File: frontend/src/routes/home/+page.server.js (hunk 3 above)
Comment (anafaraco01) [reply frontend/src/routes/home/+page.server.js:16]: Okay! Done

**Comment by v-stamenova** [comment]
Fixed the bug with the accent color

**Pull request merged**